src/explore-education-statistics-frontend/src/components/Feedback.tsx

```diff
@@ -0,0 +1,189 @@
+import FormProvider from '@common/components/form/FormProvider';
+import { Form, FormFieldTextArea } from '@common/components/form';
+import ButtonGroup from '@common/components/ButtonGroup';
+import feedbackService from '@frontend/services/feedbackService';
+import { FeedbackRequest } from '@common/services/types/feedback';
+
+export default function Feedback() {
+  const [showBanner, setShowBanner] = useState(true);
+  const [showThankYouMessage, setShowThankYouMessage] = useState(false);
+  const [showFormSection, setShowFormSection] = useState(false);
+  const [showNotUsefulForm, setShowNotUsefulForm] = useState(false);
+  const [showProblemEncounteredForm, setShowProblemEncounteredForm] =
+    useState(false);
+  const [url, setUrl] = useState('');
+  const [userAgent, setUserAgent] = useState('');
+  const [response, setResponse] = useState('');
+
+  useEffect(() => {
+    setUserAgent(navigator.userAgent);
+    setUrl(window.location.pathname);
+  }, []);
+
+  const handleUsefulFeedback = async () => {
+    await feedbackService.sendFeedback({
+      url,
+      userAgent,
+      response: 'Useful',
+    });
+
+    setShowThankYouMessage(true);
+  };
+  const showNotUseful = () => {
+    setResponse('NotUseful');
+    setShowBanner(false);
+    setShowProblemEncounteredForm(false);
+    setShowNotUsefulForm(true);
+    setShowFormSection(true);
+  };
+
+  const showProblemEncountered = () => {
+    setResponse('ProblemEncountered');
+    setShowBanner(false);
+    setShowNotUsefulForm(false);
+    setShowProblemEncounteredForm(true);
+    setShowFormSection(true);
+  };
+
+  const cancel = () => {
+    setResponse('');
+    setShowFormSection(false);
+    setShowNotUsefulForm(false);
+    setShowProblemEncounteredForm(false);
+    setShowBanner(true);
+  };
+
+  const showThankYou = () => {
+    setResponse('');
+    setShowFormSection(false);
+    setShowNotUsefulForm(false);
+    setShowProblemEncounteredForm(false);
+    setShowThankYouMessage(true);
+    setShowBanner(true);
+  };
+
+  return (
+    <div
+      className={classNames(
+        'govuk-width-container',
+        styles.responsiveContainer,
+      )}
+    >
+      {showBanner && (
+        <div className={styles.feedbackBanner}>
+          {showThankYouMessage && (
+            <div className={styles.feedbackPromptQuestions} role="alert">
+              <h3 className={styles.heading}>Thank you for your feedback</h3>
+            </div>
+          )}
+
+          {!showThankYouMessage && (
+            <>
+              <div className={styles.feedbackPromptQuestions}>
+                <h2 className={styles.heading}>Is this page useful?</h2>
+                <Button
+                  className={styles.button}
+                  variant="secondary"
+                  onClick={handleUsefulFeedback}
+                >
+                  Yes <VisuallyHidden>this page is useful</VisuallyHidden>
+                </Button>
+
+                <Button
```

Comment:
We're missing `aria-expanded` and `aria-controls` attributes that are needed so screen readers can inform the user that the form section has been opened.

Same applies for the 'Report' button below as well.